install/create_ncn_metadata_csv.md

```diff
@@ -36,12 +34,13 @@ Check the description for component names while mapping names between the SHCD a
 See [Component Names (xnames)](../operations/Component_Names_xnames.md).
 
 There are two interesting parts to the NCN metadata file:
+
 - The MAC of the BMC
-- The MAC(s) of the shasta-network interface(s)
+- The MAC(s) of the Shasta management network interface(s)
```

Comment:
Style guide says not to use (s). Either make it plural or, if it is important to emphasize that it could just be one, say "dog or dogs"

```suggestion
- The MAC(s) of the Shasta management network interfaces
```